app/views/sections/destination/DestinationWarehouseExciseView.scala.html

```diff
@@ -38,6 +41,10 @@
 
         @if(form.errors.nonEmpty) {
             @govukErrorSummary(ErrorSummaryViewModel(form))
+        } else {
+            @errorNotificationBannerHelper.content(
+                errors = DestinationWarehouseExcisePage.getMovementSubmissionErrors
+            ).map(notificationBanner(_))
```

Comment:
Currently looks like this:
![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/53828896/94c232b5-6430-4cdd-864d-39706a93e1b8)

```suggestion
            @errorNotificationBannerHelper.content(
                errors = DestinationWarehouseExcisePage.getMovementSubmissionErrors,
                hasContentHeading = false
            ).map(notificationBanner(_))
```